### ISSUE TITLE
ComboBox: Options need focus styles

### ISSUE THREAD

**Issue by lynamemi** [OfficeDev/office-ui-fabric-react#3472]
### Bug Report

- __Package version(s)__: (fill this out)
- __Browser and OS versions__: (fill this out if relevant)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No) Maybe

Requested priority: (Blocking, High, Normal, Low) Normal

Products/sites affected: (if applicable)

#### Describe the issue:

In ComboBox.styles.ts optionStyles, `getFocusStyle(theme)` is getting called, but no focus style appears in the browser. IComboBoxOptionStyles extends IButtonStyles so there maybe some conflict there?

#### Actual behavior:

No focus styles / outlines are applied to ComboBox

#### Expected behavior:

Focused options should be outlined 
![image](https://user-images.githubusercontent.com/16619843/33337512-fdd4ca82-d427-11e7-8fa7-86828e68a3f1.png)

### If applicable, please provide a codepen repro:

https://codepen.io/lynamemi/pen/LOgRRW
Open ComboBox - up and down arrows move focus, that's when the focus outline should appear.

**Comment by jspurlin** [comment]
The options don't actually need focus, they just need to be styling so that they look like they have focus (via the :after styling, like other components)

**Comment by betrue-final-final** [comment]
@lynamemi, are you still going to work on this one?

**Comment by lynamemi** [comment]
On first look, I couldn't determine why the focus styles weren't getting applied. This issue still needs ownership.

**Comment by JllyGrnGiant** [comment]
+1 on desiring a fix for this issue.

**Comment by jordandrako** [comment]
@lynamemi Now that ComboBox has been converted to css-in-js I believe, do we want to take another crack at this?

**Comment by lynamemi** [comment]
@jordandrako ComboBox was actually already converted to js styling at the time I opened this issue, however, it was converted in the old classnames way. The interesting issue here is that `getFocusStyle(theme)` is called, but not taking effect. Like I mention above, I wonder if it has to do with a conflict in IButtonStyles, but that's a stretch. 

Would definitely be good for someone more familiar with js styling (like you!) to take crack.
